packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewer.js

```diff
@@ -204,7 +204,7 @@ class LightCurveViewer extends Component {
     if (!this.isCurrentTaskValidForAnnotation()) return // Sanity check
 
     const annotations = this.annotationBrushes
-      .filter((raw) => (raw.minX !== undefined && raw.maxX !== undefined))
+      .filter((raw) => (raw.minX !== undefined && raw.maxX !== undefined && (raw.maxX - raw.minX) > 0))
       .map((raw) => {
         const x = (raw.minX + raw.maxX) / 2
         const width = (raw.maxX - raw.minX)
```

Comment:
Maybe use `Math.abs()`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs

Reply:
Are we ok with a width of zero?

Reply:
I _think_ the zeros are coming from a scenario where `maxX` and `minX` are both null, so filtering out null values should prevent this.

Reply:
It would be very hard if not impossible for someone to make a mark where subtracting min from max equals zero (max and min are the same value?), so it's more likely the values were null somehow to start with.